client-java/sql/src/main/java/org/evomaster/client/java/sql/internal/SqlHeuristicsCalculator.java

```diff
@@ -3,28 +3,96 @@
 import net.sf.jsqlparser.expression.Expression;
 import net.sf.jsqlparser.statement.Statement;
 import net.sf.jsqlparser.statement.select.FromItem;
+import net.sf.jsqlparser.statement.select.Join;
 import org.evomaster.client.java.controller.api.dto.database.schema.DbInfoDto;
+import org.evomaster.client.java.distance.heuristics.Truthness;
+import org.evomaster.client.java.distance.heuristics.TruthnessUtils;
 import org.evomaster.client.java.sql.QueryResult;
+import org.evomaster.client.java.sql.QueryResultSet;
 
-import static org.evomaster.client.java.sql.internal.SqlParserUtils.getFrom;
-import static org.evomaster.client.java.sql.internal.SqlParserUtils.getWhere;
+import java.util.Collection;
+import java.util.List;
+
+import static org.evomaster.client.java.sql.internal.SqlParserUtils.*;
 
 public class SqlHeuristicsCalculator {
 
+    public static double C = 0.1;
+    public static double C_BETTER = C + C / 2;
+    public static Truthness TRUE_TRUTHNESS = new Truthness(1, C);
+
+    private QueryResultSet queryResultSet;
+
+    private SqlHeuristicsCalculator(QueryResult[] data) {
+        final boolean isCaseSensitive = false;
```

Comment:
a final boolean given as input? why not passing `false` directly? is this for readability?

Reply:
yup, just readability